fraim/core/llms/litellm.py

```diff
@@ -4,7 +4,8 @@
 """A wrapper around litellm"""
 
 import logging
-from typing import Any, Dict, Iterable, List, Optional, Protocol, Self, Sequence, Tuple
+from collections.abc import Iterable
```

Comment:
Hmm, imports should have been sorted by `make format`. Weird that this one slipped through.